src/plugins/intel_gna/src/gna_device.hpp

```diff
@@ -123,6 +123,7 @@ class GNADeviceHelper : public GNADevice {
     void getGnaPerfCounters(std::map<std::string, InferenceEngine::InferenceEngineProfileInfo>& retPerfCounters);
     static std::string GetGnaLibraryVersion();
 
+    bool check_if_hw_available();
```

Comment:
1. Please use consistent naming style (camel case)
2. I suggest to get rid of `check_if` from the name .Exemplary names: `hwAvailable()`, `isHwAvailable()` or something similar